ts/sagas/wallet.ts

```diff
@@ -840,8 +859,38 @@ function* completionHandler(
   }
 }
 
-export function* watchWalletSaga(pagoPaClient: PagoPaClient): Iterator<Effect> {
+export function* watchWalletSaga(
+  sessionToken: SessionToken,
+  pagoPaClient: PagoPaClient
+): Iterator<Effect> {
+  // backend client for "attiva" and "verifica"
+  // TODO: replace with long timeout from https://github.com/teamdigitale/italia-app/pull/493
+  const backendClient = BackendClient(apiUrlPrefix, sessionToken);
+
+  // backend client for polling for paymentId
+  const pollingBackendClient = BackendClient(
+    apiUrlPrefix,
+    sessionToken,
+    constantPollingFetch(MAX_RETRIES_POLLING, DELAY_BETWEEN_RETRIES_MS)
+  );
+
   yield call(fetchAndStorePagoPaToken, pagoPaClient);
+
+  yield takeLatest(
```

Comment:
We should probably find a common way of handling actions. I think we can either embed all of them in the infinite loop below, or we use `takeLatest` to handle each one of them. The nice thing about `takeLatest` is that it cancels any currently executing task is the same action occurs multiple times (e.g. if multiple requests to fetch the wallets are made), the not-so-nice thing about it (admittedly, I'm fairly ignorant about it, so I'm not sure this is actually a problem) is that it might be difficult to cancel a `takeLatest` "loop" (which might be needed if, for example, the user is logged out)